### ISSUE TITLE
unnecessary wrap on BatchNorm?

### ISSUE THREAD

**Issue by johnnychen94** [DhairyaLGandhi/UNet.jl#7]
`BatchNormWrap` seems not necessary given that `Conv` requires 4D data as input:

https://github.com/DhairyaLGandhi/UNet.jl/blob/a5ac874c2b75588fb6eb6c9f855d4fbd69f9e412/src/model.jl#L1-L5

I guess I must misunderstand the idea here, can you explain it a bit why we need to unsqueeze the 4D input to 6D before feeding into BN? I didn't see this operation in other UNet implementations, for example, in [pytorch-unet](https://github.com/jvanvugt/pytorch-unet/blob/7e577223d048ace32c35c3f1e18f36cc9a8d1971/unet.py#L85-L86)
